setup.py

```diff
@@ -28,6 +28,10 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
+import sys
+if (sys.version_info < (3, 5)):
```

Comment:
```suggestion
if sys.version_info < (3, 5):
```
no need for extra parens

Reply:
I'd be inclined to keep a comment like this somewhere, but we can add that as a follow-up